.scripts/batch_release.rb

```diff
@@ -0,0 +1,44 @@
+INTEGRATIONS = %w(sentry-rails sentry-sidekiq sentry-delayed_job)
+GEMS = %w(sentry-ruby) + INTEGRATIONS
+
+def get_version_file_name(gem_name)
+  case gem_name
+  when "sentry-ruby"
+    "lib/sentry/version.rb"
+  else
+    integration_name = gem_name.sub("sentry-", "")
+    "lib/sentry/#{integration_name}/version.rb"
+  end
+end
+
+def update_version_file(gem_name, version)
+  version_file_name = get_version_file_name(gem_name)
+  file_name = "#{gem_name}/#{version_file_name}"
+  text = File.read(file_name)
+  new_contents = text.gsub(/VERSION = ".*"/, "VERSION = \"#{version}\"")
+  File.open(file_name, "w") {|file| file.puts new_contents }
+end
+
+def update_gemspec_dependency(gem_name, version)
+  file_name = "#{gem_name}/#{gem_name}.gemspec"
+  text = File.read(file_name)
+  new_contents = text.gsub(/spec.add_dependency "sentry-ruby-core", ".+"/, "spec.add_dependency \"sentry-ruby-core\", \"~> #{version}\"")
+  File.open(file_name, "w") {|file| file.puts new_contents }
+end
+
+# when craft runs scripts it inserts version to the 2nd argument
+version = ARGV[1]
```

Comment:
It also sets environment variables if that's easier:

https://github.com/getsentry/craft/blob/5030c50f8c37e81f39ce51c4a419ec787cc8f774/src/commands/prepare.ts#L249-L250

We may wanna document this.

Reply:
@BYK thanks for the tip. I'll change this later with the other version bumping script 👍